README.md

```diff
@@ -20,17 +20,38 @@
 
 Sarek is a workflow designed to run analyses on whole genome or targeted sequencing data from regular samples or tumour / normal pairs and could include additional relapses.
```

Comment:
As an outsider: this is a little vague, unless combined with the 'quote' description below the sarek logo.

What are 'regular samples'? Whole genomes of drosophila? tardigrades?

What sort of tumour?

Reply:
Good point.

What about:
> `Sarek is a workflow designed to run analyses on whole genome or targeted sequencing data.  It can also handle tumour / normal pairs and could include additional relapses.`

Reply:
> Sarek is a workflow designed to run analyses on whole genome or targeted sequencing data. It can also handle tumour / normal pairs which can optionally include additional relapses.

Might also be nice to have a second sentence pointing out that it can work with other species?

> Sarek was written to process Human and Mouse sequencing data, but much of the pipeline can be used for any species that has a reference genome.

Reply:
I think we're already going deep in the details if we try to explain more here.
We don't even specify that it's working on human to begin with.
We do have extra explanation on the genome references here: https://github.com/MaxUlysse/nf-core_sarek/blob/2.6/docs/reference.md

Reply:
ok 👌 

My thinking was that I think most people will assume that it's Human-only when there is mention of tumours and relapses, which is maybe selling Sarek a little short...